examples/ex_petstore_gorm/oas_cfg_gen.go

```diff
@@ -0,0 +1,277 @@
+// Code generated by ogen, DO NOT EDIT.
+
+package api
```

Comment:
[Commit generated files in a one separate commit please.](https://github.com/ogen-go/ogen/blob/main/CONTRIBUTING.md#committing-generated-code)